packages/fast-components-react-msft/src/pivot/pivot.props.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+import { Subtract } from "utility-types";
+import {
+    TabsHandledProps as BaseTabsHandledProps,
+    TabsItem,
+    TabsManagedClasses as BaseTabsManagedClasses,
+    TabsUnhandledProps as BaseTabsUnhandledProps,
+} from "@microsoft/fast-components-react-base";
+import {
+    ManagedClasses,
+    PivotClassNameContract,
+} from "@microsoft/fast-components-class-name-contracts-msft";
+
+export interface PivotManagedClasses extends ManagedClasses<PivotClassNameContract> {}
+export interface PivotHandledProps
+    extends PivotManagedClasses,
+        Subtract<BaseTabsHandledProps, BaseTabsManagedClasses> {
+    /**
+     * The pivots' pivot item

```

Comment:
I think this could just be "Items that will make up the pivot and pivot content"